bin/lib/Logos/Generator/MobileSubstrate/Generator.pm

```diff
@@ -19,4 +19,11 @@ sub preamble {
 	}
 }
 
+sub staticDeclarations {
+    my $self = shift;
+    return join("\n", ($self->SUPER::staticDeclarations(),
+        "asm(\".linker_option \\\"-framework\\\", \\\"CydiaSubstrate\\\"\");",
```

Comment:
```suggestion
        "asm(\".linker_option \\\"-framework\\\", \\\"CydiaSubstrate\\\"\");\n",
```
In sticking with the formatting of [dsc_extractor.cpp](https://github.com/apple-oss-distributions/dyld/blob/18d3cb0f6b46707fee6d315cccccf7af8a8dbe57/other-tools/dsc_extractor.cpp#L70)